packages/dbgpt-serve/src/dbgpt_serve/agent/app/endpoints.py

```
@@ -22,42 +25,92 @@ async def app_list(
+    )
     try:
         return Result.succ(gpts_dao.edit(gpts_app))
     except Exception as ex:
         return Result.failed(err_code="E000X", msg=f"edit app error: {ex}")
 
 
 @router.post("/v2/serve/apps")
-async def app_create(gpts_app: GptsApp):
+async def app_create(
+    gpts_app: GptsApp,
+    operator: UserRequest = Depends(require_permission("AGENT_MANAGE")),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)